helpers/internal.psm1

```diff
@@ -370,6 +370,31 @@ Function Get-DisqualifiedNetworksFromMapping {
     Return $Disqualified
 }
 
+Function Get-VDiskStatus {
+    param ( $LogFile )
+
+    $Unhealthy = $false

```

Comment:
Shouldn't we assume it's unhealthy, unless the retrieved data states that it is healthy?